src/main/java/com/minecolonies/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -331,6 +334,17 @@ private BlockPos getRandomBuilding()
     }
 
     /**
+     * Called when a guard killed an entity.
+     *
+     * @param killedEntity the entity being killed.
+     */
+    protected void onKilledEntity(EntityLivingBase killedEntity)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "killedEntity". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)